src/ui/utils/time.ts

```diff
@@ -9,3 +10,64 @@ export function formatTimestamp(ms: number) {
   const seconds = Math.round(ms / 1000.0);
   return `${Math.floor(seconds / 60)}:${padStart(String(seconds % 60), 2, "0")}`;
 }
+
+export function getDurationString(durationMs: number | null | undefined) {
```

Comment:
I don't think anything outside of the library uses this method. Why are we preserving it?